target/linux/mediatek/patches-6.1/250-pinctrl-mediatek-mt7981-add-additional-mmc-and-uart-groups.patch

```diff
@@ -0,0 +1,64 @@
+---
+ drivers/pinctrl/mediatek/pinctrl-mt7981.c | 14 ++++++++++++--
+ 1 file changed, 12 insertions(+), 2 deletions(-)
+
+--- a/drivers/pinctrl/mediatek/pinctrl-mt7981.c
++++ b/drivers/pinctrl/mediatek/pinctrl-mt7981.c
+@@ -703,6 +703,9 @@ static int mt7981_drv_vbus_funcs[] = { 1, };
+ static int mt7981_emmc_45_pins[] = { 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, };
+ static int mt7981_emmc_45_funcs[] = { 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, };
+
++static int mt7981_emmc_pins[] = { 0, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, };
```

Comment:
Please add `0` to emmc_45_pins above instead, I think this is a bug rather than actually a possible configuration (ie. without pin 0 set to function 2 already by the bootloader this never worked)

Reply:
Indeed, I added `0` to `emmc_45_pins`, and it works. Thanks for the tip.